packages/apps/docs/src/components/Layout/global.css.ts

```diff
@@ -13,5 +16,14 @@ export const globalClass = style({
     [$$pageWidth]: '1440px', //sizes
     [$$leftSideWidth]: '265px', //sizes
     [$$backgroundOverlayColor]: 'rgba(250,250,250, .8)', //colors
+    [$$borderColor]: 'rgba(112, 121, 123, 0.2)', //colors
+  },
+  selectors: {
+    [`${darkThemeClass} &`]: {
+      vars: {
+        [$$backgroundOverlayColor]: 'rgba(5,5,5, .8)', //colors
+        [$$borderColor]: 'rgba(240, 240, 240, 0.2)', //colors
```

Comment:
Don't we have these colors in react-ui package? Is it good to define it in react-ui?

Reply:
we dont. the opacity thing is specific for the documentation website atm